services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/attachment/common/AttachmentUploadServiceITest.kt

```diff
@@ -0,0 +1,98 @@
+package fi.hel.haitaton.hanke.attachment.common
+
+import assertk.all
+import assertk.assertFailure
+import assertk.assertThat
+import assertk.assertions.hasClass
+import assertk.assertions.hasMessage
+import assertk.assertions.isEqualTo
+import assertk.assertions.prop
+import fi.hel.haitaton.hanke.DatabaseTest
+import fi.hel.haitaton.hanke.attachment.USERNAME
+import fi.hel.haitaton.hanke.attachment.azure.Container
+import fi.hel.haitaton.hanke.attachment.body
+import fi.hel.haitaton.hanke.attachment.failResult
+import fi.hel.haitaton.hanke.attachment.response
+import fi.hel.haitaton.hanke.attachment.successResult
+import fi.hel.haitaton.hanke.attachment.testFile
+import fi.hel.haitaton.hanke.factory.HankeFactory
+import fi.hel.haitaton.hanke.test.Asserts.isRecent
+import okhttp3.mockwebserver.MockWebServer
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.security.test.context.support.WithMockUser
+import org.springframework.test.context.ActiveProfiles
+
+@SpringBootTest
+@ActiveProfiles("test")
+@WithMockUser(USERNAME)
+class AttachmentUploadServiceITest(
+    @Autowired private val attachmentUploadService: AttachmentUploadService,
+    @Autowired private val attachmentRepository: HankeAttachmentRepository,
+    @Autowired private val hankeFactory: HankeFactory,
+    @Autowired private val fileClient: MockFileClient
+) : DatabaseTest() {
+
+    private lateinit var mockClamAv: MockWebServer
```

Comment:
mockClamAv can be removed from HankeAttachmentServiceITests.

Reply:
I don't think it can. Uploadattachment scans the input file.

Reply:
I was talking about this, but there was no change on the line to comment on:
https://github.com/City-of-Helsinki/haitaton-backend/blob/8ea7659a520212ed3a49c3cf8b8fa79086801b24/services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/attachment/hanke/HankeAttachmentServiceITests.kt#L64

I should had been clearer. I'll remove it in #515.